cmd/umoci/unpack.go

```diff
@@ -49,7 +49,12 @@ creation with umoci-repack(1).`,
 	// unpack reads manifest information.
 	Category: "image",
 
-	Flags: []cli.Flag{},
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "keep-dirlinks",
```

Comment:
This flag should probably also be added to `raw-unpack`. And docs should be updated (it doesn't need much).

Reply:
Cool, I pushed a commit for these.

Reply:
Thanks. :D